src/thunderbots/software/ai/navigator/util.cpp

```diff
@@ -5,3 +5,19 @@ double calculateTransitionSpeedBetweenSegments(const Point &p1, const Point &p2,
 {
     return final_speed * (p2 - p1).norm().project((p3 - p2).norm()).len();
 }
+
+std::vector<MovePrimitive> convertToMovePrimitives(unsigned int robot_id,
+                                                   const std::vector<Point> &points)
+{
+    std::vector<MovePrimitive> movePrimitives;
+    movePrimitives.reserve(points.size());
+
+    for (const Point &point : points)
+    {
+        MovePrimitive movePrimitive =
+            MovePrimitive(robot_id, point, point.orientation(), 0, false, false);
```

Comment:
You don't need to specify the last two params

Reply:
Actually I think this ok. Furthermore, we should actually think about how these params will get passed from the Intent to the Primitive.

Eg. If a MoveIntent has the `dribble` boolean enabled (eg. for receiving the ball), we want the primitives to have these flags as well so we have the behavior we expect.

@jonl112 @jared-paul do you think it's also reasonable to pass a `MoveIntent` to this function? Basically, we would want the function to know what extra parameters to set, and this would be a bit more generic than just adding two boolean parameters (although that works too).

Thoughts on this?

Reply:
It's kind of weird because since it is a utility function I think it should be "only necessary information included" (don't want unnecessary information that comes from elsewhere eg. a MoveIntent object).

Do you know if this function will be used only when interacting with MoveIntents?

Reply:
Yeah I was thinking about this too. I don't like passing in `MoveIntent` and the points generated from the `MoveIntent` into same function.

Reply:
I think there might be a problem:

Let's say a robot takes a path like: (0 represents robot at point, --- represents path)
0 --- 0 --- 0 --- 0

What if we need it to activate it's kicker or dribbler at only one of these points?
If we pass 2 `bool` params or a single `MoveIntent` each `MovePrimitive` will mimic the other for enabling its dribbler or kicker. 

Right now we can only set it to false or true for all points

Do you think this is a problem or do we only care about the state of the robot at its final destination?

Reply:
I think dribbler is on during path and kicker at end of path. @MathewMacDougall is that right?

Reply:
We will lose the ability to run the dribbler/kicker at different segments if we use a single MoveIntent, but I think that will be fine. 
Generally, I think any segments that have the options selected have a different *intent* than where they are not. So I'm fine with passing a MoveIntent.

Reply:
I agree with just passing the booleans.

Regarding enabling autokick or the dribbler just for certain segments, that's not something this function should need to worry about. If the `Action` / `Navigator` wants to enable or disable the dribbler at a certain point, it would just provide a new MoveIntent with the updated parameters, so it would have control over when these things are enabled. This utility function should just need to worry about the translation.